examples/js/loaders/GLTFLoader.js

```diff
@@ -2106,20 +2106,61 @@ THREE.GLTFLoader = ( function () {
 
 						}
 
-					} else if ( primitive.mode === WEBGL_CONSTANTS.LINES ) {
+					} else if ( primitive.mode === WEBGL_CONSTANTS.LINES ||
+						primitive.mode === WEBGL_CONSTANTS.LINE_STRIP ||
+						primitive.mode === WEBGL_CONSTANTS.LINE_LOOP ) {
 
-						mesh = new THREE.LineSegments( geometry, material );
+						var cacheKey = 'LineBasicMaterial:' + material.uuid;
 
-					} else if ( primitive.mode === WEBGL_CONSTANTS.LINE_STRIP ) {
+						var lineMaterial = scope.cache.get( cacheKey );
 
-						mesh = new THREE.Line( geometry, material );
+						if ( ! lineMaterial ) {
 
-					} else if ( primitive.mode === WEBGL_CONSTANTS.LINE_LOOP ) {
+							lineMaterial = new THREE.LineBasicMaterial();
+							THREE.Material.prototype.copy.call( lineMaterial, material );
```

Comment:
Why not `lineMaterial.copy( material )`?

Reply:
Because here is trying to copy the properties between different materials.
`lineMaterial.copy( material )` will copy some `undefined` data to `lineMaterial`, for example `linewidth` because `LineBasicMaterial` had `linewidth` but `MeshStandardMaterial` doesn't have.

Reply:
Ah, right... and they'll become `undefined`...

Reply:
We'll have to clean all this some day...

Reply:
Yup, but how? Like

    `    if ( source.something !== undefined ) this.something = source.something;`

in `*Material.copy()`, that means `*Material.copy()` allows copying between different material types?

Reply:
Yeah, I think so.